features/step_definitions/web_steps.rb

```diff
@@ -241,9 +241,7 @@ def with_scope(locator)
 end
 
 When /^(?:|I )wait for ajax requests to complete/ do
-  wait_until do
-    page.evaluate_script('$.active') == 0
-  end
+  page.evaluate_script('$.active') == 0
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.